src/wrap-with-error-handling.js

```diff
@@ -1,14 +1,18 @@
-const MapDClient = (typeof window !== "undefined" && window.MapDClient) || require("../build/thrift/node/mapd.thrift.js").Client // eslint-disable-line global-require
-const TMapDException = (typeof window !== "undefined" && window.TMapDException) || require("../build/thrift/node/mapd_types.js").TMapDException // eslint-disable-line global-require
-const Thrift = (typeof window !== "undefined" && window.Thrift) || require("thrift").Thrift // eslint-disable-line global-require
+/* eslint-disable consistent-this */
```

Comment:
Same comment from before on scoping the eslint-disable and eslint-enable as tightly as possible. Perhaps also just a follow-up improvement to capture, in both cases.